rust/agents/relayer/src/settings/matching_list.rs

```diff
@@ -252,11 +285,11 @@ fn parse_addr<E: Error>(addr_str: &str) -> Result<H256, E> {
 mod test {
```

Comment:
also noticing that we don't ever test `matches`, I think a quick test case for it would be helpful.